Makefile

```
@@ -160,11 +160,30 @@ vet: ## Run go vet
+# Prefer Nx so CI finishes deterministically; seed corpora also run under plain go test.
+FUZZTIME ?= 10000x
+# Packages that define Fuzz* tests. Keep in sync when adding new fuzz targets.
+FUZZ_PACKAGES ?= ./pkg/ociclient ./pkg/mlflowclient ./internal/eval_hub/handlers ./internal/eval_hub/storage/sql/shared ./internal/eval_runtime_sidecar/handlers
+
+test: ## Run unit tests (including fuzz seed corpora and a short fuzzing pass)
 	@echo "Running unit tests..."
 	@bash -c 'set -o pipefail; go test -v ./internal/... ./cmd/... ./pkg/... | ${PWD}/scripts/grcat ${PWD}/.conf.go-test'
+	@$(MAKE) test-fuzz
 	@echo "Unit tests complete"
 
+test-fuzz: ## Run mutational fuzzing briefly for each Fuzz* test
```

checkmake (0.3.2)

[warning] Target body for "test-fuzz" exceeds allowed length of 5 lines (10).

(maxbodylength)